components/setting_item_min/setting_item_min.tsx

```diff
@@ -51,6 +51,7 @@ export default class SettingItemMin extends React.PureComponent<Props> {
 
     handleUpdateSection = (e: React.MouseEvent<HTMLElement>) => {
         e.preventDefault();
+        e.stopPropagation();
```

Comment:
wondering why this is needed 🤔

Reply:
Both the edit button and the tile use the same onClick event, when you click the edit button it register as a button click and a tile click. This will stop the click from bubbling up. The reason for keeping onClick for the edit button is because it's used to open the tile during keyboard navigation